examples/employee-app/src/main/java/com/example/helidon/employee/EmployeeRepositoryImplDB.java

```diff
@@ -0,0 +1,157 @@
+		return resultList;
+	}
+
+	@Override
+	public List<Employee> getByLastName(String name) {
+		String queryStr = "SELECT * FROM EMPLOYEE WHERE LASTNAME LIKE ?";
+		List<Employee> resultList = this.query(queryStr, name);
+		return resultList;
+	}
+
+	@Override
+	public List<Employee> getByTitle(String title) {
+		String queryStr = "SELECT * FROM EMPLOYEE WHERE TITLE LIKE ?";
+		List<Employee> resultList = this.query(queryStr, title);
+		return resultList;
+	}
+
+	@Override
+	public List<Employee> getByDepartment(String department) {
+		String queryStr = "SELECT * FROM EMPLOYEE WHERE DEPARTMENT LIKE ?";
+		List<Employee> resultList = this.query(queryStr, department);
+		return resultList;
+	}
+
+	@Override
+	public Employee save(Employee employee) {
+		String insertTableSQL = "INSERT INTO EMPLOYEE "
+				+ "(ID, FIRSTNAME, LASTNAME, EMAIL, PHONE, BIRTHDATE, TITLE, DEPARTMENT) "
+				+ "VALUES(EMPLOYEE_SEQ.NEXTVAL,?,?,?,?,?,?,?)";
+
+		try (PreparedStatement preparedStatement = this.conn.prepareStatement(insertTableSQL)) {
+
+			preparedStatement.setString(1, employee.getFirstName());
+			preparedStatement.setString(2, employee.getLastName());
+			preparedStatement.setString(3, employee.getEmail());
+			preparedStatement.setString(4, employee.getPhone());
+			preparedStatement.setString(5, employee.getBirthDate());
+			preparedStatement.setString(6, employee.getTitle());
+			preparedStatement.setString(7, employee.getDepartment());
+
+			preparedStatement.executeUpdate();
+
+		} catch (SQLException e) {
+			System.out.println("SQL Add Error: " + e.getMessage());
+
+		} catch (Exception e) {
+			System.out.println("Add Error: " + e.getMessage());
+		}
+		return employee;
+	}
+
+	@Override
+	public void deleteById(String id) {
+		String deleteRowSQL = "DELETE FROM EMPLOYEE WHERE ID=?";
+		try (PreparedStatement preparedStatement = this.conn.prepareStatement(deleteRowSQL)) {
+			preparedStatement.setInt(1, Integer.parseInt(id));
+			preparedStatement.executeUpdate();
+
+		} catch (SQLException e) {
+			System.out.println("SQL Delete Error: " + e.getMessage());
+		} catch (Exception e) {
+			System.out.println("Delete Error: " + e.getMessage());
+		}
+
+	}
+
+	@Override
+	public Employee getById(String id) {
+		String queryStr = "SELECT * FROM EMPLOYEE WHERE ID =?";
+		List<Employee> resultList = this.query(queryStr, Integer.parseInt(id));
+
+		if (resultList.size() > 0) {
+			return resultList.get(0);
+		} else {
+			return null;
+		}
+
+	}
+
+	public List<Employee> query(String sqlQueryStr, Object value) {
+		
+		List<Employee> resultList = new ArrayList<>();
+		try (PreparedStatement stmt = conn.prepareStatement(sqlQueryStr)) {
+			if (value!=null){
+				if (value instanceof String){
+					stmt.setString(1, value+"%");
+				}else {
+					stmt.setInt(1,(Integer) value);
+				}
+			}
+			ResultSet rs = stmt.executeQuery();
+			while (rs.next()) {
+				resultList.add(new Employee(rs.getString("ID"), rs.getString("FIRSTNAME"), rs.getString("LASTNAME"),

```

Comment:
indentation seems to be incorrect here.

Reply:
Reformatted with NetBeans. Should be fixed now.